app/assets/stylesheets/helpers/_button.scss

```diff
@@ -0,0 +1,15 @@
+// TODO: We can move this into GOV.UK Publishing Components if it makes sense to do so.
+
+@import "design-patterns/buttons"; // Elements Button import globally depends on mixins defined in here.
+@import "elements/helpers"; // Elements Button import globally depends on mixins defined in here.
+// DEBT/RISK: Includes a .example-highlight-grid global CSS helper
+// https://github.com/alphagov/govuk_elements/blob/874e287/public/sass/elements/_helpers.scss#L12
+// and a reimplementation of .visually-hidden
+// https://github.com/alphagov/govuk_elements/blob/874e287/public/sass/elements/_helpers.scss#L36
+@import "elements/buttons";
```

Comment:
I originally did as you described but it felt really odd to bring across legacy styles when we have fairly self contained versions of the buttons in GOV.UK Elements.

That said I did this in mind of moving this into static when it's needed so I would be okay doing as you've proposed.